lua/neo-tree/ui/renderer.lua

```diff
@@ -621,6 +621,8 @@ M.position = {
         _, state.position.node_id = pcall(node.get_id, node)
       end
     end
+    local win_state = vim.fn.winsaveview()
+    state.position.topline = win_state.topline
```

Comment:
@ghostbuster91 I've notice recently that when I use the `:Neotree current` command from a terminal, the neotree window will usually be scrolled down so that the last line is the top line. I don't have time to look into it right now, but I would suspect that it is the result of this change.

Reply:
My guess would be that the above two lines should be pushed up into the block above:
```lua
      if success and node then
        _, state.position.node_id = pcall(node.get_id, node)
        local win_state = vim.fn.winsaveview()
        state.position.topline = win_state.topline
      end
```